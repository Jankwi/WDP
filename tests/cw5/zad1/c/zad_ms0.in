10
-9 6 3 7 2 1 6 8 2 8