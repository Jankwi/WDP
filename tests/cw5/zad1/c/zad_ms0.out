34
25
37
24
43
34
25
18
43
18
